src/main/java/ledger/user_interface/ui_controllers/window/LoginPageController.java

```
@@ -112,8 +122,13 @@ private void login() {
         try {
             DbController.INSTANCE.initialize(this.filePath, pwd);
             Startup.INSTANCE.newStage(new Scene(new MainPageController()), "Ledger");
+            Preferences preferences = Preferences.userRoot().node("LedgerSoftware");
+            preferences.put(LAST_DATABASE_FILE_KEY,this.filePath);
+            preferences.flush();
         } catch (StorageException e) {
             this.setupErrorPopup("Unable to connect to database", e);
+        } catch (BackingStoreException e){
+
```

Empty catch blocks are bad...